cms/templates/sudo/third_party_auth_links.html

```diff
@@ -0,0 +1,17 @@
+{% load i18n third_party_auth_links %}
+<script type="text/javascript">
+    function thirdPartySignin(event, url) {
+      event.preventDefault();
+      window.location.href = url;
+    }
+</script>
+
+<span class="deco-divider">
+    <span class="copy">{% trans "or" %}</span>
+</span>
+<div class="form-actions form-third-party-auth">
+
+{% for enabled in providers %}
+    <button type="submit" class="button button-primary button-{{enabled.provider_id}} login-{{enabled.provider_id}}" {% if enabled.has_account %} onclick="thirdPartySignin(event, '{{pipeline_url|get_pipeline_url:enabled.provider_id}}');" {% else %} disabled="disabled" {% endif %} ><span class="icon fa {{enabled.icon_class}}"></span>{% trans 'Sign in with' %} {% trans enabled.name %}</button>
```

Comment:
Concatenating translated strings like this doesn't provide sufficient context to our translators. Please check out our i18n guide on how to design translated strings with proper context: https://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#don-t-append-strings-interpolate-values